components/board/numato/board.go

```diff
@@ -384,6 +418,21 @@ func connect(ctx context.Context, name resource.Name, conf *Config, logger loggi
 		path = devs[0].Path
 	}
 
+	// Find the numato board's productid
+	products := getSerialDevices()
+
+	var productID int
+	for _, product := range products {
+		if product.ID.Vendor != 0x2a19 {
+			continue
+		}
+		productID = product.ID.Product
```

Comment:
This is going to get the last valid productID, no matter what. If I have 2 numato boards plugged into my laptop with different productIDs, one of them is going to have bad data here. Is there a way to tie these to the `SerialPath` in the config?

Reply:
I didn't think we allowed more than one numato board plugged in due to the error on line 414.

Reply:
Very good point! Put in a comment somewhere around here explaining that it's okay to get the last one because we only support one plugged in. 

...and maybe put a `break` just after the current line, so we find the first numato board and then don't bother looking through all the other USB devices after that. but that's such a small optimization it might not be worth the effort.